### PR TITLE
added checks for friends and valid entities

### PR DESCRIPTION
**Pull request by SwiftHD**
3rd times the charm <3

### CODE REVIEW

File: Horion/Module/Modules/TriggerBot.cpp [hunk 1]
```diff
@@ -16,20 +17,35 @@ const char* TriggerBot::getModuleName()
...
+			return;
+
 		if (localPlayer->getEntityTypeId() == target->getEntityTypeId())
 		{
 			localPlayer->swingArm();
 			gm->attack(target);
 		}
-		Odelay = 0;	
+		Odelay = 0;
+		}
```
Comment (richardletshacks) [inline Horion/Module/Modules/TriggerBot.cpp]: remove this bracket, you dont need it as you removed the else

Comment (SwiftHD) [reply Horion/Module/Modules/TriggerBot.cpp]: ffs i removed it on VS but forgot to upload

**Review by richardletshacks — APPROVED** [review]
Looks good. @SurvirvorHRG please merge

File: Horion/Module/Modules/TriggerBot.cpp [hunk 2]
```diff
@@ -16,21 +17,35 @@ const char* TriggerBot::getModuleName()
...
 	}
-}
```
Comment (ghost) [inline Horion/Module/Modules/TriggerBot.cpp:34]: uhm

File: Horion/Module/Modules/TriggerBot.cpp (hunk 1 above)
Comment (ghost) [reply Horion/Module/Modules/TriggerBot.cpp]: adding a closing bracket here and removing one from the bottom isnt a great way to fix the situation

**Review by ghost — CHANGES_REQUESTED** [review]

File: Horion/Module/Modules/TriggerBot.cpp (hunk 2 above)
Comment (SwiftHD) [reply Horion/Module/Modules/TriggerBot.cpp:34]: IT WORKS
as long as their is the correct amount of closing brackets it doesn't matter

Comment (SwiftHD) [reply Horion/Module/Modules/TriggerBot.cpp:34]: Happy now? added bottom back just so it looks nice.

**Pull request merged**